assets/src/features/home-page/components/VideoroomHomePage.tsx

```diff
@@ -60,56 +50,67 @@ const VideoroomHomePage: React.FC = () => {
 
   return (
     <HomePageLayout>
-      <section className="flex flex-col items-center gap-y-12 sm:gap-y-18">
+      <section className="flex flex-col items-center gap-y-12 sm:justify-around sm:gap-y-0">
         {deviceManager.errorMessage && (
           <div className="w-full bg-red-700 p-1 text-white">{deviceManager.errorMessage}</div>
         )}
         <div className="flex flex-col items-center gap-y-6 text-center">
           <h2 className="text-3xl sm:text-5xl">Videoconferencing for everyone</h2>
           <p className="font-aktivGrotesk text-xl">Join the existing room or create a new one to start the meeting</p>
         </div>
```

Comment:
This feels too big compared to the rest or the video and the rest are a bit too low -the proportions seem different from the design

Reply:
Do you mean the font size? Because it is exactly the same as in the Figma design

Reply:
Not sure what's the source of the problem, but I mean the area with red arrow is too large, while the one marked with black arrow - too narrow
![image](https://user-images.githubusercontent.com/2119556/218053889-de6cdc7a-ba66-45e1-9823-53b7f82e8726.png)


Here's a design screen for comparison
![image](https://user-images.githubusercontent.com/2119556/218053378-1f19f897-dddb-4e74-95f3-aa2aac205720.png)